openapi/events-api.yaml

```diff
@@ -2497,6 +2501,8 @@ components:
       title: Legacy Labels
       description: |-
             Legacy labels to further categorise the event.
+
+            This field will only be present if you have access to it in your subscription.
```

Comment:
Perhaps add: New accounts from April 2025 will not have access to the legacy labels field.